libraries/standard/http/src/http_client.c

```diff
@@ -54,6 +55,58 @@ static HTTPStatus_t _addHeader( HTTPRequestHeaders_t * pRequestHeaders,
                                 const char * pValue,
                                 size_t valueLen );
 
+/**
+ * @brief Receive HTTP response from the transport recv interface.
+ *
+ * @param[in] pTransport Transport interface.
+ * @param[in] pResponse Response buffer.
+ * @param[in] bufferLen Length of the response buffer.
+ *
+ * @return Returns #HTTP_SUCCESS if successful. If there was a network error or
+ * more bytes than what was specified were read, then #HTTP_NETWORK_ERROR is
+ * returned.
+ */
+HTTPStatus_t _receiveHttpResponse( const HTTPTransportInterface_t * pTransport,
```

Comment:
(Nit) I think `_readResponse` is a more appropriate name

Reply:
The other functions say _send I think this is more consistent. We are receiving from the transport interface anyways.